LandSandBoat/server
# [SQL] Correct Astral Flow flags and Valid target for Carbuncle and Leviathan

**_I affirm:_**

## What does this pull request do?


Corrects my mistake related to Astral Flow flags on Carbuncle and Leviathan. This push was missed during the initial rework of Carbuncle and a resemblance mistake on my part for Leviathan. Also worth mentioning, invalid flags set on these skills will cause the client to crash/force close. I have ensured and tested this change to function as intended

## Steps to test these changes


level 99 SMN
!godmode
!reset
Use Astral Flow while having Carbuncle or Leviathan summoned. See the ability correctly preform the requested action.
![image](https://github.com/LandSandBoat/server/assets/98081508/693ed1d5-0ecf-4e37-9edf-cb7bf5a33a41)
![image](https://github.com/LandSandBoat/server/assets/98081508/57f7db6a-1b84-421f-80ec-74e0f611ccca)
